src/main/resources/icons/io.kestra.plugin.documentdb.svg

```diff
@@ -1,18 +1,4 @@
-    </g>
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Could you use this SVG one for both icons instead please?

![Image](https://github.com/user-attachments/assets/25a9565a-3373-4dc8-b24c-4af346783c4e)